golang/cosmos/x/swingset/module.go

```diff
@@ -149,9 +154,9 @@ func (am AppModule) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) []abci.V
 	return []abci.ValidatorUpdate{}
 }
 
-func (am AppModule) checkSwingStoreExportSetup() {
+func (am *AppModule) checkSwingStoreExportSetup() {
 	if am.swingStoreExportDir == "" {
-		panic(fmt.Errorf("SwingStore export dir not set"))
+		am.swingStoreExportDir = "/tmp/swingset_export"
```

Comment:
What is the motivation for this change? This is used by genesis import/export which should set this appropriately.

Reply:
I don't know the motivation, but I see that @schnetzlerjoe made the change.  Perhaps this panic was being triggered by the x/vtransfer `ibc_middleware_test.go`?